testing/src/lib.rs

```diff
@@ -561,31 +562,34 @@ pub enum ConsensusTestError {
     TooManyFailures,
 }
 
-/// An implementation to make the trio `NETWORK`, `STORAGE` and `STATE` implement [`NodeImplementation`]
+/// An implementation of [`NodeImplementation`] for testing.
 pub struct TestNodeImpl<
     TYPES: NodeType,
     LEAF: LeafType<NodeType = TYPES>,
     PROPOSAL: ProposalType<NodeType = TYPES, Election = ELECTION>,
+    VOTE: VoteType<TYPES>,
     NETWORK,
     STORAGE,
     ELECTION,
 > {
     _pd_0: PhantomData<TYPES>,
     _pd_1: PhantomData<LEAF>,
     _pd_2: PhantomData<PROPOSAL>,
-    _pd_3: PhantomData<NETWORK>,
-    _pd_4: PhantomData<STORAGE>,
-    _pd_5: PhantomData<ELECTION>,
+    _pd_3: PhantomData<VOTE>,
+    _pd_4: PhantomData<NETWORK>,
```

Comment:
nit:
```rust
  pd: PhantomData<(TYPES, LEAF, PROPOSAL, VOTE, STORAGE, ELECTION)>
```
might be simpler to parse

Reply:
Yes! I updated all places where multiple `PhantomData` were used.